test/integrationTests/lspInlayHints.integration.test.ts

```diff
@@ -3,20 +3,16 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as path from 'path';
 import * as vscode from 'vscode';
-
 import * as jestLib from '@jest/globals';
-import testAssetWorkspace from '../../omnisharptest/omnisharpIntegrationTests/testAssets/testAssetWorkspace';
-import * as path from 'path';
-import { activateCSharpExtension, isRazorWorkspace, isSlnWithGenerator } from './integrationHelpers';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+import * as integrationHelpers from './integrationHelpers';
 import { InlayHint, InlayHintKind, Position } from 'vscode-languageserver-protocol';
 
-const describeif = (condition: boolean) => (condition ? jestLib.describe : jestLib.describe.skip);
-const skip = !isRazorWorkspace(vscode.workspace) && !isSlnWithGenerator(vscode.workspace);
-
-describeif(skip)(`LSP Inlay Hints ${testAssetWorkspace.description}`, function () {
-    let fileUri: vscode.Uri;
+jestLib.describe(`LSP Inlay Hints ${testAssetWorkspace.description}`, function () {
     jestLib.beforeEach(async function () {
+        await testAssetWorkspace.restoreLspToolsHostAsync();
```

Comment:
we weren't restoring before so the project wasn't technically loading correctly.  this fixes that.  At some point we need some tests to verify that projects are all actually loading (with all their references)